.github/workflows/ci.yml

```diff
@@ -31,6 +45,8 @@ jobs:
     if: startsWith(github.ref, 'refs/heads/bump-version-')
     needs: [test]
     runs-on: ubuntu-latest
+    env:
+      COMPONENT: mackerel-container-agent
```

Comment:
Is this needed to be an environment variable?
It seems that just setting `role-session-name: "mackerel-container-agent-${{ github.run_id }}-${{ github.run_number }}"` is much simpler.

Reply:
fixed on aeab8b2312ee4401bdff558c31856684d1845686